lib/features/building_detail_view/presentation/widgets/building_data_source_link.dart

```diff
@@ -0,0 +1,32 @@
+import "package:flutter/widgets.dart";
+import "package:flutter_riverpod/flutter_riverpod.dart";
+
+import "../../../../theme/app_theme.dart";
+import "../../../../utils/context_extensions.dart";
+
+class BuildingDataSourceLink extends ConsumerWidget {
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return Text.rich(
+      TextSpan(
+        text: "${context.localize.data_come_from_website}: ",
+        style: const TextStyle(
+          fontWeight: FontWeight.bold,
+        ),
+        children: [
+          TextSpan(
+            text: "www.przewodnik.pwr.edu.pl",
```

Comment:
This could be moved to l10n package

Reply:
I think it's better to add this to UrlConfig file

Reply:
Ofc, my typo here